src/js/renderShabad.js

```diff
@@ -22,7 +22,10 @@ function renderShabad(gurbani, nav) {
     .concat(prefs.shabadToggles)
     .forEach(option => document.getElementById(option).click());
 
-  Object
+    var larivaarStatus = localStorage.getItem('shabadToggles').indexOf('larivaar-toggle')>-1;
```

Comment:
Is `shabadToggles` a preference? In that case we should use `getPref` method to avoid conflicts in future.